Navigation-Examples/Constants.swift

```diff
@@ -104,5 +104,19 @@ let listOfExamples: [NamedController] = [
         controller: BuildingExtrusionViewController.self,
         storyboard: nil,
         pushExampleToViewController: true
+    ),
+    (
+        name: "Predictive Caching",
+        description: "Demonstrates how to use predictive caching for navigation.",
+        controller: PredictiveCachingViewController.self,
+        storyboard: nil,
+        pushExampleToViewController: true
```

Comment:
```suggestion
        pushExampleToViewController: false
```

to prevent issues related to `UIViewController` dismissal.

Reply:
Thanks for the suggestion, I'll fix this part.